config/configs_basic.yaml

```diff
@@ -313,17 +312,6 @@ report_preprocessing:
     outlier_charts: False
     source_path: "NA"
 
-report_generation:
-  master_path: 'report_stats'
-  id_col: 'ifa'
-  label_col: 'income'
-  corr_threshold: 0.4
-  iv_threshold: 0.02
-  drift_threshold_model: 0.1
-  dataDict_path: 'data/income_dataset/data_dictionary.csv'
-  metricDict_path: 'data/metric_dictionary.csv'
-  final_report_path: 'report_stats'
-
```

Comment:
This part is needed to generated the basic report as well. Any reasons why we are deleting this?

Reply:
See above, it is not possible to run this with the currently available PyPI release of Anovos due to an excessive amount of memory being used.

Reply:
The same as above, suggested to make another PR to better document it.

Reply:
See above.